package.json

```diff
@@ -7,16 +7,17 @@
     "browserify": "^11.1.0",
     "chai": "^3.2.0",
     "mocha": "^2.3.2",
+    "popper": "0.0.20",
     "sinon": "^1.16.1",
     "sinon-chai": "^2.8.0"
   },
   "scripts": {
-    "test": "mocha",
+    "test-node": "mocha",
     "clean": "rm -rf lib",
-    "compile": "mkdir -p lib && browserify src/rhumb.js --standalone rhumb -o lib/rhumb.js",
+    "compile": "npm run clean && mkdir lib && browserify src/rhumb.js --standalone rhumb -o lib/rhumb.js",
```

Comment:
For some reason, your `mkdir -p lib` was creating a `-p` and `lib` folder for me. Probably because Windows..